views/data.html

```diff
@@ -0,0 +1,76 @@
+{% extends "layouts/one-column.html" %}
+
+{% block breadcrumb %}
+  <li><a href="/">Home</a></li>
+  <li>{{ title | truncate(35) }}</li>
+{% endblock %}
+
+{% block main %}
+  <article>
+    <section>
+      <h2>Overview</h2>
+      <p>
+        The entire OpenTrials database is saved monthly automatically. This
+        is what we use to power our API, and contains everything that you can
+        see on this website. Having the entire database in your local machine
+        allows you to analyze the data using your favorite tool, or even create
+        your own copy of the OpenTrials API.
+      </p>
+      <p>
+        As with any usage of OpenTrials's data, make sure you read and
+        understand our <a href="/terms-of-use">Terms of use</a>.
+      </p>
+    </section>
+    <section>
+      <h2>Download</h2>
+      {% if dumps.length > 0 %}
+        <p>
+          Below are the latest available dumps of our database. Their
+          file names contain the date at which the dump was produced:
+        </p>
+        <ul>
+          {% for dump in dumps | sort(true, false, 'label') %}
```

Comment:
An alternative way of displaying would be to generate a table and parse the date from the file names to use as a column. Easier on the eye to find the most or the least recent dump available. But if the number of dumps stays small, we're better off with the plain list..